core/object/script_language.h

```diff
@@ -431,10 +431,12 @@ class ScriptLanguage : public Object {
 		uint64_t call_count;
 		uint64_t total_time;
 		uint64_t self_time;
+		uint64_t internal_time;
 	};
 
 	virtual void profiling_start() = 0;
 	virtual void profiling_stop() = 0;
+	virtual void profiling_set_save_native_calls(bool opt) = 0;
```

Comment:
```suggestion
	virtual void profiling_set_save_native_calls(bool p_opt) = 0;
```
Arguments are prefixed with `p_`